app/javascript/packages/verify-flow/verify-flow-step-indicator.tsx

```diff
@@ -0,0 +1,73 @@
+import { StepIndicator, StepIndicatorStep, StepStatus } from '@18f/identity-step-indicator';
+import { t } from '@18f/identity-i18n';
+
+// i18n-tasks-use t('step_indicator.flows.idv.getting_started')
+// i18n-tasks-use t('step_indicator.flows.idv.verify_id')
+// i18n-tasks-use t('step_indicator.flows.idv.verify_info')
+// i18n-tasks-use t('step_indicator.flows.idv.verify_phone_or_address')
+// i18n-tasks-use t('step_indicator.flows.idv.secure_account')
+
+/**
+ * Mapping of flow form steps to corresponding step indicator step.
+ */
+const FLOW_STEP_STEP_MAPPING: Record<string, string> = {
+  personal_key: 'secure_account',
+  personal_key_confirm: 'secure_account',
+};
+
+/**
+ * Sequence of step indicator steps.
+ */
+const STEP_INDICATOR_STEPS: string[] = [
+  'getting_started',
+  'verify_id',
+  'verify_info',
+  'verify_phone_or_address',
+  'secure_account',
+];
```

Comment:
Would it be useful to make this an array of enum types?

```suggestion
type StepIndicatorStep = 'getting_started' | 'verify_id' | 'verify_info' | 'verify_phone_or_address' | 'secure_account';
const STEP_INDICATOR_STEPS: StepIndicatorStep[] = [
  'getting_started',
  'verify_id',
  'verify_info',
  'verify_phone_or_address',
  'secure_account',
];
```

Reply:
Updated in d16b70405.